nowcasting_dataset/data_sources/fake.py

```diff
@@ -172,32 +173,65 @@ def topographic_fake(batch_size, image_size_pixels):
     return Topographic(xr_dataset)
 
 
+def add_uk_centroid_osgb(x, y):
+    """Add a OSGB value to make coords in center of UK"""
+    lat = np.random.uniform(51, 55)
+    lon = np.random.uniform(-2.5, 1)
+
+    x_center, y_center = lat_lon_to_osgb(lat=lat, lon=lon)
+
+    x = x + x_center
+    y = y + y_center
+
+    return x, y
+
+
+def make_random_point_coords_osgb(size: int):
+    """Make random coords [OSGB] for pv site, of gsp"""
+    # this is about 100KM
+    x = np.random.randint(0, 10 ** 5, size)
+    y = np.random.randint(0, 10 ** 5, size)
+
+    return add_uk_centroid_osgb(x, y)
+
+
+def make_random_image_coords_osgb(size: int):
+    """Make random coords for image. These are ranges for the pixels"""
+    x = 4 * 10 ** 3 * np.array((range(0, size)))
```

Comment:
what's this bit?

```
4 * 10 ** 3
```

could we do a similar variable thing as above here or is it pretty random anyways?